contracts/src/v0.8/interfaces/FlagsDevInterface.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface FlagsDevInterface {
+  function getFlag(address) external view returns (bool);
```

Comment:
Not sure if Interfaces follow this style guide: https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/interfaces/FlagsInterface.sol

Reply:
@kaleofduty wdyt? IMHO we should keep consistent and pick one style for all.

Reply:
I'm happy with both options here. I have to say that the style guide leads to very long files, which is kind of suboptimal for readability for an interface, but I also see your argument about consistency.

Since we're short on time here, my suggestion would be to just leave this file as is.

Reply:
Ok, leaving it as is for now and adding an "open question" to our Solidity style guide.